thirdeye/thirdeye-frontend/app/pods/components/rootcause-metrics/component.js

```diff
@@ -125,22 +155,32 @@ export default Ember.Component.extend({
     }
   ),
 
-  changesFormatted: Ember.computed(
-    'changes',
-    function () {
-      const { changes } = this.getProperties('changes');
-      return Object.keys(changes).reduce((agg, urn) => {
-        const value = changes[urn];
-        const sign = value > 0 ? '+' : '';
-        if (Math.abs(value) > 5) {
-          agg[urn] = 'spike';
-        } else {
-          agg[urn] = sign + (value * 100).toFixed(2) + '%';
-        }
+  _changesOffset(offset) {
```

Comment:
Since there's a computed property called `changesOffset` above, can we give this a different name?

Reply:
fixed, plus comments